Source/WebCore/animation/WebAnimation.idl

```diff
@@ -54,6 +54,8 @@ typedef (double or CSSNumericValue) CSSNumberish;
     attribute AnimationTimeline? timeline;
     [ImplementedAs=bindingsStartTime] attribute CSSNumberish? startTime;
     [ImplementedAs=bindingsCurrentTime] attribute CSSNumberish? currentTime;
+    [ImplementedAs=bindingsRangeStart, EnabledBySetting=ScrollDrivenAnimationsEnabled] attribute (TimelineRangeOffset or CSSNumericValue or CSSKeywordValue or DOMString) rangeStart;
+    [ImplementedAs=bindingsRangeEnd, EnabledBySetting=ScrollDrivenAnimationsEnabled] attribute (TimelineRangeOffset or CSSNumericValue or CSSKeywordValue or DOMString) rangeEnd;
```

Comment:
No need for `ImplementedAs` here since we don't need to do anything specific for when the values are set through the bindings. This means that the getter and setter on `WebAnimation` should remove the `Bindings` bit in their name.

Reply:
Fixed.